gradle/wrapper/gradle-wrapper.properties

```diff
@@ -1,5 +1,6 @@
+#Wed Sep 11 10:09:54 IST 2019
 distributionBase=GRADLE_USER_HOME
 distributionPath=wrapper/dists
-distributionUrl=https\://services.gradle.org/distributions/gradle-5.1.1-all.zip
 zipStoreBase=GRADLE_USER_HOME
 zipStorePath=wrapper/dists
+distributionUrl=https\://services.gradle.org/distributions/gradle-5.4.1-all.zip
```

Comment:
Are we also upgrading the gradle plugin version?

Reply:
We should update the `complieSdkVersion`, `targetSdkVersion` and the `buildToolsVersion` too. This is a different issue though. 
A separate issue and pr should be created for this.

Reply:
@yashk2000 Shall I take this?